cli/cmd/install/install.go

```diff
@@ -232,8 +275,13 @@ timescaledb-single:
 		fmt.Println("skipping the wait for pods to come to a running state because --skip-wait is enabled.")
 	}
 
-	fmt.Println("The Observability Stack has been installed successfully")
-	fmt.Println(release.Info.Notes)
+	if release.Info != nil {
+		fmt.Println(release.Info.Notes)
```

Comment:
nit: it might be a good idea to swap `fmt` to some propper logger in the future.